operator/backupcontroller/executor.go

```diff
@@ -60,7 +61,8 @@ func (b *BackupExecutor) listAndFilterPVCs(ctx context.Context, annotation strin
 
 	pods := &corev1.PodList{}
 	pvcPodMap := make(map[string]corev1.Pod)
-	if err := b.Config.Client.List(ctx, pods, client.InNamespace(b.backup.Namespace)); err != nil {
+	labelselector, _ := labels.Parse("!" + job.K8uplabel);
+	if err := b.Config.Client.List(ctx, pods, client.InNamespace(b.backup.Namespace), client.MatchingLabelsSelector{labelselector}); err != nil {
```

Comment:
The lint job is complaining here to not use an unkeyed field in the `client.MatchingLabelsSelector` literal.

Reply:
fixed